packages/calling/src/CallHistory/CallHistory.ts

```diff
@@ -139,6 +139,14 @@ export class CallHistory extends Eventing<CallHistoryEventTypes> implements ICal
         }
       }
     );
+    this.sdkConnector.registerListener<CallSessionEvent>(
+      MOBIUS_EVENT_KEYS.LOCUS_CALL_SESSION_EVENT,
+      async (event?: CallSessionEvent) => {
+        if (event && event.data.userSessions.userSessions) {
```

Comment:
I think this will given an error in some edge case, if data.userSessions is undefined

Reply:
```suggestion
        if (event && event.data.userSessions && event.data.userSessions.userSessions) {
```

Reply:
I believe data.userSessions will not be undefined because it will always return an array which can be empty but the object always has some value. Hence we don't need to add a check for it.